code/game/objects/structures/holosign.dm

```diff
@@ -200,7 +205,7 @@
 	if(!shockcd)
 		if(ismob(user))
 			var/mob/living/M = user
```

Comment:
You'll have to unindent the contents
``user`` is already pathed as ``/mob/living``
```suggestion
	if(shockcd || !isliving(user))
	    return
```